app/models/partner.rb

```diff
@@ -28,11 +28,12 @@ class Partner < ApplicationRecord
 
   include DiaperPartnerClient
   after_create :update_diaper_partner
-
+  # better to extract this outside of the model
```

Comment:
This is a great point -- I'll create an issue to refactor this behavior out into a concern; we have multiple models doing this.

Reply:
would love to get this ticket @armahillo  <3